web-scraper/README.md

```diff
@@ -1,70 +1,32 @@
+## Features
+Web Scraper is a generic easy-to-use [actor](https://apify.com/actors) for crawling arbitrary web pages
 and extracting structured data from them using a few lines of JavaScript code.
 The actor loads web pages in the Chromium browser and renders dynamic content.
 Web Scraper can either be configured and run manually in a user interface, or programmatically using the API.
 The extracted data is stored in a dataset, from where it can be exported to various formats,
 such as JSON, XML, or CSV.
 
-If you're not familiar with web scraping or front-end web development in general,
-you might prefer to start
-with [**Web scraping tutorial**](https://apify.com/docs/scraping/web-scraper-tutorial)
-in Apify documentation,
-and then continue with [**Scraping with Web Scraper**](https://apify.com/docs/scraping/tutorial/web-scraper),
-a tutorial which will walk you through all the steps and provide a number of examples.
-
-## Table of content
-
-<!-- toc -->
-
-- [Usage](#usage)
-- [Limitations](#limitations)
-- [Input configuration](#input-configuration)
-  - [Run mode](#run-mode)
-  - [Start URLs](#start-urls)
-  - [Link selector](#link-selector)
-  - [Pseudo-URLs](#pseudo-urls)
-  - [Page function](#page-function)
-    - [**`customData: Object`**](#customdata-object)
-    - [**`enqueueRequest(request, [options]): AsyncFunction`**](#enqueuerequest-request-options-asyncfunction)
-    - [**`env: Object`**](#env-object)
-    - [**`getValue(key): AsyncFunction`**](#getvalue-key-asyncfunction)
-    - [**`globalStore: Object`**](#globalstore-object)
-    - [**`input: Object`**](#input-object)
-    - [**`jQuery: Function`**](#jquery-function)
-    - [**`log: Object`**](#log-object)
-    - [**`request: Object`**](#request-object)
-    - [**`response: Object`**](#response-object)
-    - [**`saveSnapshot(): AsyncFunction`**](#savesnapshot-asyncfunction)
-    - [**`setValue(key, data, options): AsyncFunction`**](#setvalue-key-data-options-asyncfunction)
-    - [**`skipLinks(): AsyncFunction`**](#skiplinks-asyncfunction)
-    - [**`underscoreJs: Object`**](#underscorejs-object)
-    - [**`waitFor(task, options): AsyncFunction`**](#waitfor-task-options-asyncfunction)
-- [Proxy configuration](#proxy-configuration)
-- [Results](#results)
-- [Additional resources](#additional-resources)
-- [Upgrading](#upgrading)
-
-<!-- tocstop -->
-
-## Usage
-
-To get started with Web Scraper,
-you only need two things. First, tell the scraper which web pages
+If you're not familiar with web scraping, you might like to go through the following resources before using Web Scraper:
+- First read our [Beginner's Guide to Web Scraping](https://apify.com/web-scraping).
+- Read our short step-by-step tutorial on [How to scrape any web page for beginners](https://blog.apify.com/how-to-scrape-any-website-for-beginners/).
+- Then go over the [Web scraping tutorial](https://apify.com/docs/scraping/web-scraper-tutorial) in the Apify documentation.
```

Comment:
This was wrong even before, but should link here https://docs.apify.com/tutorials/apify-scrapers/getting-started